src/main/java/frc/trigon/robot/poseestimation/apriltagcamera/AprilTagCamera.java

```diff
@@ -0,0 +1,199 @@
+package frc.trigon.robot.poseestimation.apriltagcamera;
+
+import edu.wpi.first.math.Matrix;
+import edu.wpi.first.math.VecBuilder;
+import edu.wpi.first.math.geometry.*;
+import edu.wpi.first.math.numbers.N1;
+import edu.wpi.first.math.numbers.N3;
+import frc.trigon.robot.Robot;
+import frc.trigon.robot.constants.FieldConstants;
+import frc.trigon.robot.poseestimation.poseestimator.PoseEstimator6328;
+import org.littletonrobotics.junction.Logger;
+import org.photonvision.PhotonUtils;
+
+/**
+ * An april tag camera is a class that provides the robot's pose, from a camera.
+ * An april tag is like a 2D barcode used to find the robot's position on the field.
+ * By using the angle and size of the tag that the camera sees, it can estimate its position.
```

Comment:
Since the tag's position on the field is known, we can calculate our position relative to it, therefore estimating our position on the field.